### DISCUSSION TITLE
persistent: true + cache: true (Learning)

### DISCUSSION THREAD

**Discussion by connorjs** [vercel/turborepo#4327] [Help] [answered]
Curious/Learning question: What’s a valid use case for `persistent: true` and `cache: true`?

_Motivation_: Every time I write `persistent: true`, it feels tedious/duplicative to write `cache: false`. I can understand concern about `persistent` implying `cache: false` (changing the default), but if a `cache: true` use case exists, I would grumble less at least 😃

**Comment by mehulkar** [comment, marked as answer]
You are correct in pointing this out! There isn't a good case for caching persistent tasks. We talked about implicitly making persistent tasks `cache: false`, but ultimately decided this was opaque behavior that we didn't want to force onto the user without making it clear what was happening. 

Since shipping persistent tasks, we've also shipped [Run Summaries](https://turbo.build/repo/docs/troubleshooting#check-the-run-summary) and [Resolved Task Configurations in Dry Run output](https://turbo.build/repo/docs/troubleshooting#task-configuration), so it's a little more likely that we can add implicit behavior like this.

Feel free to open up an issue (or PR) and I can take it to the team!
